src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "order": 660,
+  "icon": "icon.png",
+  "ports": [
+    {
+      "name": "HttpPort",
+      "useHttps": false
+    },
+    {
+      "name": "HttpsPort",
+      "useHttps": true
+    }
+  ],
+  "disableHttpsSymbol": "NoHttps",
+  "symbolInfo": [
+    {
+      "id": "Hosted",
+      "isVisible": "true"
+    },
+    {
+      "id": "PWA",
+      "isVisible": "true"
+    }
+  ]
```

Comment:
Just confirming you're seeing this as expected in VS -> New Project menu?

Reply:
Yes, all of these are being shown as 3 options (Configure for HTTPS, ASP .NET Core Hosted, Progressive Web Application) when creating a new "Blazor WebAssembly App Empty" project.